### PR TITLE
[release] merging crate-release-update into crate

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Fixed the `gtk-sys` version update.

File: Cargo.toml [hunk 1]
```diff
@@ -35,37 +35,34 @@ default = ["v3_0"]
...
+gdk-sys = "0.9.1"
+glib = "0.9.0"
+glib-sys = "0.9.1"
+gdk-pixbuf = "0.8.0"
+gtk-source-sys = "0.9.1"
+gdk-pixbuf-sys = "0.9.1"
+pango = "0.8.0"
+gio = "0.8.0"
+gio-sys = "0.9.1"
+gtk = "0.8.0"
```
Comment (EPashkin) [inline Cargo.toml]: 0.9.1

Comment (EPashkin) [reply Cargo.toml]: Sorry, I wrong

Comment (GuillaumeGomez) [reply Cargo.toml]: ;)

**Pull request merged**